CHANGELOG.md

```diff
@@ -77,6 +77,7 @@
 
 ### Chore & Maintenance
 
+- `[micromatch]` Increase peer dependency of `micromatch` to `^4.0.7` ([#15082](https://github.com/jestjs/jest/pull/15082))
```

Comment:
```suggestion
- `[*]` Increase version of `micromatch` to `^4.0.7` ([#15082](https://github.com/jestjs/jest/pull/15082))
```

I'm not allowed to edit your PR, otherwise I'd apply the change myself 🙂

Reply:
@SimenB Thank you for your feedback. I have applied the suggestion you mentioned. Could you please check if everything is now in order?